src/data/index.js

```diff
@@ -65,9 +72,18 @@ function updateStatsMap(key, matchedMeta, statsMap) {
  * @param {import('./fetchData').SensorEntry} sEntry
  */
 function loadRegularSignal(sEntry, meta, timeMap, statsMap) {
-  const matchedMeta = meta.epidata.find(
-    (d) => d.data_source === sEntry.id && d.signal === sEntry.signal && (!d.time_type || d.time_type === 'day'),
-  );
+  // find the matching meta data by looping through the candidates and fallback to the first one
+  const baseFilter = (d) =>
+    d.data_source === sEntry.id && d.signal === sEntry.signal && (!d.time_type || d.time_type === 'day');
+  const candidates = regularSignalMetaDataGeoTypeCandidates.slice();
+  let matchedMeta = null;
+  while (!matchedMeta && candidates.length > 0) {
```

Comment:
Nit: (Optional to fix) this is a bit hard to read, consider a single expression, something like:

`meta.epidata.filter((d)=>baseFilter(d)).sort(sortByArrayOrder)[0]` where sortByArrayOrder chooses "msa" then "county" otherwise "state" in that order, or choose a data structure that has an explicit order between levels.

Reply:
something like in the latest update?